ci/test/03_before_install.sh

```diff
@@ -6,15 +6,20 @@
 
 export LC_ALL=C.UTF-8
 
+FOLD_COMMAND=""
+if [[ ${TRAVIS} == "true" ]]; then
+  FOLD_COMMAND="travis_fold:"
+fi
```

Comment:
Not sure if it is worth it to keep travis related code in our ci files, given that we move away from it.

Reply:
./ci/test/ doesn't use travis anymore